Plugins/Flow.Launcher.Plugin.Program/Main.cs

```diff
@@ -203,16 +198,7 @@ private void DisableProgram(IProgram programToDelete)
                 _win32s.FirstOrDefault(x => x.UniqueIdentifier == programToDelete.UniqueIdentifier)
                     .Enabled = false;
```

Comment:
please append a `?` here for nullreference since we are using `FirstOrDefault`. Though, shall we use `First` instead?🤔

Reply:
Use `First` instead. We've checked it exists.